core/src/test/scala/org/apache/spark/deploy/PythonRunnerSuite.scala

```diff
@@ -64,4 +65,31 @@ class PythonRunnerSuite extends SparkFunSuite {
     intercept[IllegalArgumentException] { PythonRunner.formatPaths("hdfs:/some.py,foo.py") }
     intercept[IllegalArgumentException] { PythonRunner.formatPaths("foo.py,hdfs:/some.py") }
   }
+
+  test("PythonErrorUtils should have corresponding methods for default methods in SparkThrowable") {
+    // Find default methods in SparkThrowable
+    val defaultMethods = classOf[SparkThrowable]
+      .getMethods
+      .filter(m => m.getDeclaringClass == classOf[SparkThrowable] && m.isDefault)
```

Comment:
We don't need to filter with the default method here? We can always call methods through `PythonErrorUtils` and it should have the same methods as `SparkThrowable`?